hack/update-deps.sh

```diff
@@ -21,19 +21,24 @@ FLOATING_DEPS=(
 )
```

Comment:
I think you need to change above to have VERSION and @VERSION?

Reply:
yes, also had forgotten the gomod line. still working...